templates/web/tfl/report/new/roads_message.html

```diff
@@ -0,0 +1,6 @@
+<div id="js-roads-responsibility" class="box-warning hidden">
```

Comment:
Does/will this need adding to all other London borough cobrands, e.g. bexley? Are they showing bus stop/traffic light assets? Don't think they are at present.